### ISSUE TITLE
Bug - Accessibility – Home page - One link does not receive focus state correctly.

### ISSUE THREAD

**Issue by JamesONeill-DoF** [dof-dss/architecture-blog#28]
Home page - One link does not receive focus state in the same manner as others on this page: 'accessibility guidance from Government Digital Service'.

**Comment by JamesONeill-DoF** [comment]
Re-tested. This link now receives focus state as expected (and in the same way as others on this page). Closing.
